code/game/objects/items/storage/pill_packets.dm

```diff
@@ -2,71 +2,86 @@
 
 /obj/item/storage/pill_bottle/packet
 	name = "Pill packet"
-	desc = "Containts pills. Once you take them out they don't go back in."
-	greyscale_config = null //this line stops pill packets from turning into pill bottles
-	icon_state = "pill_packet"
+	desc = "Contains pills. Once you take them out they don't go back in."
+	icon_state = "packet_canister"
 	cant_hold = list(/obj/item/reagent_containers/pill) //Nada. Once you take the pills out. They don't come back in.
-	storage_slots = 4
+	storage_slots = 8
 	max_w_class = 0
-	max_storage_space = 4
+	max_storage_space = 8
+	greyscale_config = null //So that we get packets and not pill bottles.
```

Comment:
this is already null, unneeded var

Reply:
It's not already null

Reply:
greyscale_config = /datum/greyscale_config/pillbottle
From the parent /obj/item/storage/pill_bottle

Reply:
If I don't null it, the packets turn into bottles